database/schema.js

```diff
@@ -0,0 +1,18 @@
+var pgp = require( 'pg-promise' )()
```

Comment:
This code that is duplicated between this file and the `database/dbBookStore.js` file stands out to me.  To prevent maintenance issues later - What happens if the local connection string changes, for example?  You would have to update in two places, remembering that this string exists in two places.

Consider consolidating this functionality into the `dbBookStore.js` module by exporting the `db` object:
```js
/*** database.dbBookStore.js ***/
const pgp = require( 'pg-promise' )()
const connectionString = process.env.DATABASE_URL || 'postgress://localhost:5432/guarded-porcupine-db'
const db = pgp( connectionString )

module.exports = db
```
Now, we can import this `db` object in modules that need it:
```js
/*** database/schema.js ***/
const db = require( './dbBookStore' )
// etc.
```

Finally, consider renaming this file (`schema.js`) into something a little more intention revealing - I associate the name "schema" with a sql file that defines the *structure* of a database.  This file is a js file that *loads/creates* the structure of the database.